client/src/main/scala/com/dwolla/cloudflare/domain/model/ratelimits/RateLimitAction.scala

```diff
@@ -1,10 +1,26 @@
 package com.dwolla.cloudflare.domain.model.ratelimits
 
-case class RateLimitAction (
-  mode: String,
-  timeout: Int,
-  response: Option[RateLimitActionResponse] = None
-)
+sealed trait RateLimitAction {
+  val mode: String
+  val timeout: Option[Int]
+  val response: Option[RateLimitActionResponse]
+}
+
+case class TimeoutRateLimitAction(mode: String, timeoutSeconds: Int, response: Option[RateLimitActionResponse] = None) extends RateLimitAction {
+  override val timeout: Option[Int] = Some(timeoutSeconds)
```

Comment:
It feels a little weird to have the timeout be provided as an `Int` in the constructor and then converted to an `Option[Int]` here.

Reply:
This should go away if we make the DTO match the ADT structure.